packages/node/build.js

```diff
@@ -34,6 +34,10 @@ async function main() {
       '@vercel/build-utils',
       '-e',
       'typescript',
+      '-e',
+      'ts-morph',
+      '-e',
+      'path-to-regexp',
       '-o',
```

Comment:
Now that I look at it, this needs to excluded all `dependencies`.

Lets do the same loop we use in the cli package here:

https://github.com/vercel/vercel/blob/b52df7a533160f605f5584e2e5401e3701297ade/packages/cli/scripts/build.ts#L46-L54

Reply:
Done!